src/devtools/client/debugger/src/components/TestInfo/TestStepItem.tsx

```diff
@@ -95,8 +95,21 @@ export function TestStepItem({
         });
 
         if (consolePropsProperty?.object) {
+          const consolePropsPauseData = await client.getObjectWithPreview(
+            consolePropsProperty.object,
+            pauseResult.pauseId
+          );
+          const consoleProps = consolePropsPauseData.objects?.find(
+            o => o.objectId === consolePropsProperty.object
+          );
+
+          if (consoleProps?.preview) {
+            // suppress the prototype entry in the properties output
+            consoleProps.preview.prototypeId = undefined;
```

Comment:
:hand: I think this is technically mutating the cache entry. Probably better to make a defensive nested copy here.

Reply:
It's probably safe for this object in this pause point but agree we should be defensive here